src/cart/crt_group.c

```diff
@@ -1090,6 +1091,8 @@ crt_grp_priv_create(struct crt_grp_priv **grp_priv_created,
 
 	grp_priv->gp_size = 0;
 	grp_priv->gp_refcount = 1;
+	grp_priv->gp_psr_rank = CRT_NO_RANK;
+
```

Comment:
how does setting it to no rank fix the issue ? the title says look up of rank 0, is rank 0 set to CRT_NO_RANK ?